transport/http/jsonrpc/server.go

```diff
@@ -11,6 +11,8 @@ import (
 	httptransport "github.com/go-kit/kit/transport/http"
 )
 
+const requestIDKey = "request-id"
```

Comment:
Since this is just a string with a relatively common value, it could conflict with other keys injected into the context. [See the documentation for context.WithValue](https://golang.org/pkg/context/#WithValue).

```suggestion
type requestIDKeyType struct{}

var requestIDKey requestIDKeyType
```